README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/TDycores-Project/TDycore.svg?branch=master)](https://travis-ci.org/TDycores-Project/TDycore)
+[![Build Status](https://github.com/TDycores-Project/TDycore/workflows/auto_pr_test/badge.svg)
```

Comment:
How is this not malformed? Seems you need a link target.

Reply:
Ah, right. The symbol soup and the long turnaround time got me. Will double check. Thanks!

Reply:
Apparently it wasn't malformed--the first [ was just ignored, and the rest of the URI was correct. Too much to ask for consistent syntax in Markdown. Fixed now.